weblate/checks/chars.py

```diff
@@ -39,6 +39,22 @@
 FRENCH_PUNCTUATION = {";", ":", "?", "!"}
 
 
+class AcceleratorKeyCheck(TargetCheck):
+    """Check for inconsistent accelerator keys."""
+
+    check_id = "accelerators"
+    name = _("Accelerator key")
+    description = _("Source and translation do not both contain an accelerator key")
```

Comment:
For opt-in behavior:

```suggestion
    description = _("Source and translation do not both contain an accelerator key")
    default_disabled = True
```